src/backend/base/langflow/services/settings/base.py

```diff
@@ -241,6 +241,14 @@ def set_database_url(cls, value, info):
         if not value:
             logger.debug("No database_url provided, trying LANGFLOW_DATABASE_URL env variable")
             if langflow_database_url := os.getenv("LANGFLOW_DATABASE_URL"):
+                if not langflow_database_url.startswith("postgres://") and not langflow_database_url.startswith(
+                    "postgresql://"
+                ):
+                    if langflow_database_url.startswith("sqlite://"):
+                        logger.warning("Sqlite has been abandoned, please use postgres.")
```

Comment:
I don't think we can say this. SQLite is still going to be used by most users that run Langflow locally.